src/selectors/characterSelectors.js

```diff
@@ -1 +1,20 @@
+import { wordsToRemoveEnum } from "@/enums/wordsToRemoveEnum"
+
 export const characters = (state) => state.character.characters
+
+export const getCharacterById = (state, id) => {
+  const characters = state.character.characters
+
+  const character = characters.find((character) => character.id === id)
+  if (!character) return null
+
+  const filteredCharacter = {}
+  Object.keys(character).forEach((key) => {
+    const value = character[key]
+    filteredCharacter[key] = wordsToRemoveEnum.words.includes(value)
+      ? ""
+      : value
+  })
```

Comment:
What is going on here? All this function should be doing is 
` return state.character.characters.find((character) => character.id === id)`